trying to open a project after installing 1307

Notes
======
trying to open a project after installing Travis build 1307 (Android 4.4.2)

Environment
======
Environment Key | Value
:----: | :----:
version | 11.0
build | 164
UDID | f710314c4d27668b
Android Release | 4.4.2
Android SDK | 19
Brand | rockchip
Device | rk292x
Stack trace
======
```java
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.translationstudio.androidapp/com.door43.translationstudio.ui.translate.TargetTranslationActivity}: java.lang.NullPointerException
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2195)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2245)
	at android.app.ActivityThread.access$800(ActivityThread.java:135)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1196)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:136)
	at android.app.ActivityThread.main(ActivityThread.java:5017)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:795)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:611)
	at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
	at com.door43.translationstudio.ui.translate.TargetTranslationActivity.onCreate(TargetTranslationActivity.java:222)
	at android.app.Activity.performCreate(Activity.java:5231)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2159)
	... 11 more

```

--- a/app/src/main/java/com/door43/translationstudio/ui/translate/TargetTranslationActivity.java
+++ b/app/src/main/java/com/door43/translationstudio/ui/translate/TargetTranslationActivity.java
@@ -235,8 +235,6 @@ public class TargetTranslationActivity extends BaseActivity implements ViewModeF
             public void onClick(View v) {
                 removeSearchBar();
                 openTranslationMode(TranslationViewMode.READ, null);
-
-                TargetTranslationActivity activity = (TargetTranslationActivity) v.getContext();
             }
         });
 
@@ -245,8 +243,6 @@ public class TargetTranslationActivity extends BaseActivity implements ViewModeF
             public void onClick(View v) {
                 removeSearchBar();
                 openTranslationMode(TranslationViewMode.CHUNK, null);
-
-                TargetTranslationActivity activity = (TargetTranslationActivity) v.getContext();
             }
         });
 
@@ -257,8 +253,6 @@ public class TargetTranslationActivity extends BaseActivity implements ViewModeF
                 mMergeConflictFilterEnabled = false;
                 setMergeConflictFilter(mMergeConflictFilterEnabled);
                 openTranslationMode(TranslationViewMode.REVIEW, null);
-
-                TargetTranslationActivity activity = (TargetTranslationActivity) v.getContext();
             }
         });
 

